dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -234,10 +235,12 @@ public void evaluate() throws Throwable {
                          * This sleep is arbitrary, if we continue to have problems, a longer sleep could be tried. Or a new solution and remove
                          * this retry.
                          */
-                        if (!expectedFFDCs.isEmpty() && !errors.isEmpty()) {
+                        if (!expectedFFDCs.isEmpty() && !errors.isEmpty() && (i + 1 <= retryCount)) {
```

Comment:
instead of doing an extra `i` check here why not add the Thread.sleep(retryInterval) to the latter half of the for loop. 

```
for (int i = 1, retryInterval = 120; i <= retryCount; i++, Thread.sleep(retryInterval), retryInterval *= 10)
```

You'd have to check to make sure this works.  Haven't worked much with multivariable for loops in java.

Reply:
Your way would add an extra extraneous sleep at the end without then entering the loop to check again, so I'm inclined to leave it as is.

Reply:
Ohhh you are totally right.  👍🏼 I missed that part.